observability/monitoring/configuring-user-workload-monitoring/configuring-metrics-uwm.adoc

```diff
@@ -31,6 +31,8 @@ include::modules/monitoring-example-remote-write-authentication-settings.adoc[le
 
 include::modules/monitoring-example-remote-write-queue-configuration.adoc[leveloffset=+2,tags=**;!CPM;UWM]
 
+include::modules/monitoring-table-of-remote-write-metrics.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
 .Additional resources
```

Comment:
Same consideration about the additional resources section, which you can ignore if it was intentional